ext/Logger.php

```diff
@@ -91,33 +119,40 @@ public function afterTest(TestEvent $e)
 
     public function endTest(TestEvent $e)
     {
-        $this->logger->info("PASSED");
+        self::info("PASSED");
+        self::$logger = self::getDefaultLogger();
     }
 
     public function testFail(FailEvent $e)
     {
-        $this->logger->alert($e->getFail()->getMessage());
-        $this->logger->info("# FAILED #");
+        self::alert($e->getFail()->getMessage());
+        self::info("# FAILED #");
     }
 
     public function testError(FailEvent $e)
     {
-        $this->logger->alert($e->getFail()->getMessage());
-        $this->logger->info("# ERROR #");
+        self::alert($e->getFail()->getMessage());
+        self::info("# ERROR #");
     }
 
     public function testSkipped(FailEvent $e)
     {
-        $this->logger->info("# Skipped #");
+        self::info("# Skipped #");
     }
 
     public function testIncomplete(FailEvent $e)
     {
-        $this->logger->info("# Incomplete #");
+        self::info("# Incomplete #");
     }
 
     public function beforeStep(StepEvent $e)
     {
-        $this->logger->info((string) $e->getStep());
+        self::info((string)$e->getStep());
+    }
+
+    public static function __callStatic($name, $arguments)
```

Comment:
Hi @DavertMik, thanks for taking a look - do you mean reverse this change - https://github.com/Codeception/Codeception/pull/3654/commits/cbc56494a85b1fa742a4c225cc51db4402ac21d1

I was trying to get rid of wrapping the functions, but I can see why you might not like that (SOLID).